tutorials/viewports/viewports.rst

```diff
@@ -180,6 +180,12 @@ You can also set the :ref:`Viewport <class_Viewport>` to use HDR, HDR is very us
 
 If you know how the :ref:`Viewport <class_Viewport>` is going to be used, you can set its Usage to either 3D or 2D. Godot will then
 restrict how the :ref:`Viewport <class_Viewport>` is drawn to in accordance with your choice; default is 3D.
+The 2D usage mode is slightly faster compared to the 3D one; it's a good idea to set it to 2D if your viewport doesn't render anything in 3D.
```

Comment:
And uses less memory! Actually memory is the better reason to use a 2D Viewport over a 3D one.